packages/tc-schema-sdk/data-accessors/relationship-types.js

```diff
@@ -0,0 +1,32 @@
+const type = require('./type');
+
+module.exports = {
+	// todo move to object rest parameters for options when upgrading node
+	accessor(options = {}) {
+		const types = this.rawData.getTypes();
+		const relationshipTypes = this.rawData.getRelationshipTypes();
+
+		// Find relationships from type properties
+		return types.reduce((relationships, { name, properties }) => {
+			Object.entries(properties).forEach(
+				([propName, { relationship, type: typeName }]) => {
+					if (
+						relationship ||
```

Comment:
I do wonder if we should expect type properties that reference relationship types to specify `isRelationship: true`. It would simplify the logic here (and maybe elsewhere?), and might make the schema easier to understand (or maybe not - I'm really not sure)

On the other hand, it's more future proof if we expect less in the yaml files now, and can add it in later (does not need to be a breaking change if we're careful about rolling out library updates before actually changing the schema)

What do you think?

Reply:
I agree, and if we assign `isRelationship: true` at each type properties, it should be done at `getType()`, I think.

However, now we're using `isRelationship` field in here - https://github.com/Financial-Times/treecreeper/blob/master/packages/tc-schema-sdk/data-accessors/type.js#L143 so i'll try this logic being simply with carefully. Probably above logic indicates that the property type is relationship explicitly so we can use it.